master/buildbot/steps/master.py

```diff
@@ -38,18 +38,18 @@ class MasterShellCommand(BuildStep):
     description = 'Running'
     descriptionDone = 'Ran'
     descriptionSuffix = None
-    renderables = ['command', 'env', 'workdir']
+    renderables = BuildStep.renderables + ['command', 'env']
```

Comment:
Yes it does. See accumulate class list

Le ven. 16 janv. 2015 13:46, Mikhail Sobolev <notifications@github.com> a
écrit :

> In master/buildbot/steps/master.py
> <https://github.com/buildbot/buildbot/pull/1483#discussion-diff-23078146>:
>
> > @@ -38,18 +38,18 @@ class MasterShellCommand(BuildStep):
> >      description = 'Running'
> >      descriptionDone = 'Ran'
> >      descriptionSuffix = None
> > -    renderables = ['command', 'env', 'workdir']
> > +    renderables = BuildStep.renderables + ['command', 'env']
>
> I need to check whether it really accumulates them automatically.
>
> In any case, a seemingly simple change caused a test case to error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1483/files#r23078146>.
>